app/grants/views.py

```diff
@@ -532,12 +534,41 @@ def grant_fund(request, grant_id, grant_slug):
             })
 
     splitter_contract_address = settings.SPLITTER_CONTRACT_ADDRESS
+    
+    # handle phantom funding
+    active_tab = 'normal'
+    fund_reward = None
+    round_number = 3
+    can_phantom_fund = request.user.is_authenticated and request.user.groups.filter(name='phantom_funders').exists()
+    phantom_funds = PhantomFunding.objects.filter(profile=request.user.profile, grant=grant, round_number=round_number)
+    is_phantom_funding_this_grant = can_phantom_fund and request.user.is_authenticated and phantom_funds.exists()
```

Comment:
extra auth check here already checked when `can_phantom_fund` is created

Reply:
updating